libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.m

```diff
@@ -354,9 +354,9 @@ - (void)start {
         return;
     }
     
-    [self willChangeValueForKey:@"executing"];
```

Comment:
As per the "... Upon starting the operation, your start method should also update the execution state of the operation as reported by the executing property. You do this by sending out KVO notifications for the __executing__ key path, which lets interested clients know that the operation is now running. Your executing property must also provide the status in a thread-safe manner.", should you leave the keypath as __executing__ in the start method?